sim/battle.ts

```diff
@@ -447,13 +447,21 @@ export class Battle extends Dex.ModdedDex {
 			((b.priority || 0) - (a.priority || 0)) ||
 			((b.speed || 0) - (a.speed || 0)) ||
 			-((b.subOrder || 0) - (a.subOrder || 0)) ||
+			((a.thing && b.thing) ? -(b.thing.abilityOrder - a.thing.abilityOrder) : 0) ||
```

Comment:
Do you remember why this was added? It seems to have caused the deterministic speed ties I've been trying to fix:

https://github.com/smogon/pokemon-showdown/pull/6319#issuecomment-579372632